docs/src/components/home/DesignSystemComponents.tsx

```diff
@@ -0,0 +1,268 @@
+      },
+    },
+    [theme.breakpoints.up('md')]: {
+      marginTop: -1,
+      '&:not(:nth-of-type(3n + 1))': {
+        marginLeft: -1,
+      },
+      '&:first-of-type': {
+        borderTopLeftRadius: theme.shape.borderRadius,
+      },
+      '&:last-of-type': {
+        borderBottomRightRadius: theme.shape.borderRadius,
+        borderStyle: 'dashed',
+      },
+      '&:nth-last-of-type(3)': {
+        borderBottomLeftRadius: theme.shape.borderRadius,
+      },
+      '&:nth-of-type(3)': {
+        borderTopRightRadius: theme.shape.borderRadius,
+      },
+    },
+  },
+}));
+
+function Demo({
+  name,
+  children,
+  control,
+}: {
+  name: string;
+  children: React.ReactElement;
+  control?: { prop: string; values: Array<string> };
+}) {
+  const [propValue, setPropValue] = React.useState(control ? control.values[0] : '');
+  return (
+    <Box sx={{ display: 'flex', flexDirection: 'column', height: '100%' }}>
+      {control ? (
+        <Box sx={{ minHeight: 40, ml: -1, mt: -1 }}>
+          <Tabs
+            value={propValue}
+            onChange={(event, value) => setPropValue(value)}
+            sx={{
+              minHeight: 'initial',
+              '& .MuiTabs-indicator': {
+                bgcolor: 'transparent',
+                '&:before': {
+                  height: '100%',
+                  content: '""',
+                  display: 'block',
+                  width: (theme) => `calc(100% - ${theme.spacing(2)})`,
+                  bgcolor: 'primary.main',
+                  position: 'absolute',
+                  top: 0,
+                  left: (theme) => theme.spacing(1),
+                },
+              },
+              '& .MuiTab-root': {
+                px: 1,
+                pt: 0.5,
+                minWidth: 'initial',
+                minHeight: 'initial',
+                fontWeight: 'medium',
+              },
+            }}
+          >
+            {control.values.map((value) => (
+              <Tab value={value} label={value} />
+            ))}
+          </Tabs>
+        </Box>
+      ) : null}
+      <Box sx={{ flexGrow: 1, display: 'flex', alignItems: 'center', justifyContent: 'center' }}>
+        {React.cloneElement(children, {
+          ...(control && {
+            [control.prop]: propValue,
+          }),
+        })}
+      </Box>
+      <Typography fontWeight="bold">{name}</Typography>
+    </Box>
+  );
+}
+
+const DesignSystemComponents = () => {
+  const [anchor, setAnchor] = React.useState<HTMLElement | null>(null);
+  return (
+    <Container sx={{ py: 8 }}>
+      <Typography variant="body2" color="primary" fontWeight="bold" sx={{ mb: 1 }}>
+        What do you get?
+      </Typography>
+      <Typography variant="h2" sx={{ mb: 4, maxWidth: 500 }}>
+        Simple, accessible, declaritive <GradientText>components</GradientText>.
+      </Typography>
+      <Grid>
+        <div>
+          <Demo name="Button" control={{ prop: 'size', values: ['small', 'medium', 'large'] }}>
```

Comment:
> ['small', 'medium', 'large']
Reminder for the tabs here to start with uppercase -> Small, Medium, Large.

Reply:
I have doubt on this one, do we need to make it uppercase? I feel it is better to use the real value which are lowercase. @oliviertassinari any thought?

Reply:
It is about this section.

![Uploading Screen Shot 2564-07-30 at 09.03.35.png…]()

Reply:
It looks very weird to me all lowercase. Not sure why it would make sense to leave it lowercase.

Reply:
I would personally use an uppercase. In the Figma kit, we use the same convention as in the code, but in the specific context of the homepage, it doesn't feel like we hare this much into code:

<img width="222" alt="Capture d’écran 2021-08-04 à 15 22 57" src="https://user-images.githubusercontent.com/3165635/128188460-a9ad6bea-70d7-4f88-a0a9-8f874dc76b71.png">